doc/node-test-commit-matrix.md

```diff
@@ -41,8 +41,6 @@ This is assumed correct as of the date of last commit. If you notice a discrepan
     - ppcbe-ubuntu1404 (Node < 8)
```

Comment:
Probably on your backlog, but this also looks like it's retired. Same with some old Windows vs2013 stuff from a quick find in files

Reply:
VS2013 is still supported for building add-ons with Node.js v8. I think we have a policy of keeping the machines around for 6 months after the release lines that use them become EOL, so VS2013 can be removed in July 2020. I may scale it down to only 1 machine though.

Reply:
Spun up #1997 to remove the ubuntu PPCBE, but not sure if it's right